pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -214,7 +216,7 @@ cpdef convert_to_timedelta64(object ts, object unit):
 
 @cython.boundscheck(False)
 @cython.wraparound(False)
-def array_to_timedelta64(object[:] values, unit='ns', errors='raise'):
+def array_to_timedelta64(object[:] values, unit='None', errors='raise'):
```

Comment:
don't use string None, just None

Reply:
Sure .this slipped out in one of the merges.